scripts/zaru.py

```diff
@@ -0,0 +1,1288 @@
+#!/usr/bin/env python3
+#
+# Copyright 2023 Linaro
+# Copyright 2025 Antmicro
+# Copyright 2025 Analog Devices
+#
+# SPDX-License-Identifier: Apache-2.0
+
+import argparse
+import sys
+
+try:
+    import bt2
+except ImportError:
+    sys.exit(
+        "Python3 babeltrace2 module is missing. Please install it.\n"
+        "On Ubuntu, install with 'apt-get install python3-bt2'."
```

Comment:
This seems to be very limiting.

What about users not using the Ubuntu provided Python3 installation, but having a custom Python installed ?
Or users with other Linux distributions: https://docs.zephyrproject.org/latest/develop/getting_started/installation_linux.html
or mac / windows ?

Reply:
that would be better.

At least it shows that while ubuntu might be the well-tested platform, then it could work on other systems as well.